kmesh_compile_env_pre.sh

```diff
@@ -14,11 +14,11 @@ function install_libboundscheck() {
 function dependency_pkg_install() {
     if command -v apt > /dev/null; then
 	    # apt install 
-	    apt-get update && apt-get install -y git make clang libbpf-dev llvm linux-tools-generic protobuf-compiler libprotobuf-dev libprotobuf-c-dev protobuf-c-compiler cmake golang
```

Comment:
maybe remove it later, when we support compile before running

Reply:
Another installation method should be provided here. apt install could not match go 1.21 version.